### DISCUSSION TITLE
Sort apps & and create rule

### DISCUSSION THREAD

**Discussion by MayitoPG** [tnodir/fort#225] [Ideas]
Can a feature be implemented that allows to sort app by Net usage and create a rule from Statistics window? or simply  add a column 'Net Usage' on Program that would be nice

**Comment by tnodir** [comment]
See also #208

**Reply by MayitoPG** [reply]
Yeah, that's the concept. This feature will easily identify any unfamiliar o even common process that's misusing the network and allow to quickly blocking it from access to the network .

**Comment by tnodir** [comment]
#226 created
